### PR TITLE
Add PPA install instructions to README

### PR DESCRIPTION
**Pull request by Wiener234**
# Pull Request Template

## Type of change

- [ ] Bug fix
- [x] Feature
- [x] Documentation update

## Description

*I build a debian package and it is hosted in this PPA "https://github.com/Wiener234/ani-cli-ppa".

I added the install instructions with the GPG Key to the README under the Debian section with.*

## Checklist

- [ ] any anime playing
- [ ] bumped version
- [x] next, prev and replay work
- [x] quality works
- [x] downloads work
- [x] quality works with downloads
- [x] select episode -a and rapid resume work
- [x] syncplay -s works
- [x] autoplay, aka range selection, works

## Additional Testcases

- The safe bet: One Piece
- Episode 0: Saenai Heroine no Sodatekata ♭
- Unicode: Saenai Heroine no Sodatekata ♭
- Not uploaded: one piece dub episode 590
- Unreleased: soredemo ayumu wa yosetekuru
- Short id (for decryption): Log Horizon episode 1-2

### CODE REVIEW

**Review by CoolnsX — APPROVED** [review]

**Comment by Derisis13** [comment]
I'll close #789 in favour of this. I'll also write a full guide for rpm, and I'd like to see similar ones for the AUR, termux, scoop and brew (at least)

**Comment by Derisis13** [comment]
This is what I could find. Instructions for brew are missing from v2.2's readme and I can't get my head around the scoop instructions + I'm not even sure if it'd work...

File: README.md [hunk 1]
```diff
@@ -80,6 +84,39 @@ For more info on providers, please refer to [this](https://github.com/pystardust
 
 ### Linux
 
+### Debian
+
+```
+wget -qO- https://Wiener234.github.io/ani-cli-ppa/KEY.gpg | sudo tee /etc/apt/trusted.gpg.d/ani-cli.asc
+wget -qO- https://Wiener234.github.io/ani-cli-ppa/ani-cli-debian.list | sudo tee /etc/apt/sources.list.d/ani-cli-debian.list
+sudo apt update
```
Comment (port19x) [inline README.md:92]: You still need to install it...

File: README.md [hunk 2]
```diff
@@ -80,6 +84,39 @@ For more info on providers, please refer to [this](https://github.com/pystardust
...
+```
+
+### Fedora
+
+To install mpv (and vlc) you need _RPM Fusion free_ enabled. Simply follow the instructions here: https://rpmfusion.org/Configuration
+To be able to install syncplay, you'll need to enable this copr repo: https://copr.fedorainfracloud.org/coprs/batmanfeynman/syncplay/
```
Comment (port19x) [inline README.md:98]: Is this necessary information?

File: README.md [hunk 3]
```diff
@@ -80,6 +84,39 @@ For more info on providers, please refer to [this](https://github.com/pystardust
...
+
+There are currently two packages for Fedora: `ani-cli` which is a minimal installation that can only use mpv and `ani-cli-full` which contains all the functionality (note that vlc and syncplay are still optional dependencies).
```
Comment (port19x) [inline README.md:100]: Why/how?

File: README.md [hunk 4]
```diff
@@ -80,6 +84,39 @@ For more info on providers, please refer to [this](https://github.com/pystardust
...
+
+### Arch
+
+Also consider ani-cli-git
+
+```sh
+yay -S ani-cli
+```
+
+### Installing from source
```
Comment (port19x) [inline README.md:118]: Maybe give nix os a section too, they have a package thats well maintained iirc

**Review by port19x — CHANGES_REQUESTED** [review]

**Comment by port19x** [comment]
Can we port this PR to include recent readme changes?
I do find ppa & copr instructions valuable

File: README.md (hunk 3 above)
Comment (Derisis13) [reply README.md:101]: We no longer have two packages, only one that has it all.

File: README.md (hunk 4 above)
Comment (Derisis13) [reply README.md:119]: Ok. Do we know who the nixpkg maintainer is?

**Comment by Derisis13** [comment]
I'll try to work on this. I'd also like scoop added too, since it's once again maintained

**Review by port19x — APPROVED** [review]
Looks mergable too me.
Suitable merge commit message: `docs: more distrospecific instructions`
I'll wait for another sign off

**Pull request merged**